mod/settings.php

```diff
@@ -1174,7 +1174,7 @@ function settings_content(App $a) {
 
 	if (strlen(get_config('system','directory'))) {
 		$profile_in_net_dir = replace_macros($opt_tpl,array(
-			'$field' 	=> array('profile_in_netdirectory', t('Publish your default profile in the global social directory?'), $profile['net-publish'], '', array(t('No'),t('Yes'))),
+			'$field' 	=> array('profile_in_netdirectory', t('Publish your default profile in the global social directory?'), $profile_in_directory, t("Depending on the server settings your profile may be visible in public."), $profile['net-publish'], '', array(t('No'),t('Yes'))),
```

Comment:
Also, there are no `$profile_in_directory` variable declared in the file, did you mean `$publish` (declared line 425)?